.vscode/launch.json

```diff
@@ -4,214 +4,25 @@
 		{
 			"type": "node",
 			"request": "launch",
-			"name": "Gulp Build",
```

Comment:
IIRC, we are keeping these entries here to avoid merge conflicts, not sure whether we really want to make this change

Reply:
I don't really see the point in trying to merge this file. There's so much stuff that is only needed for VS Code and we've already had to make all sorts of other changes to work with our stuff that it just doesn't seem worth the effort considering the dev impact of having a hard-to-understand launch tasks menu. 

 It's easy enough to just see what they added/changed and take it in if we want it.

Reply:
sounds good to me